tests/suites/os/suite.js

```diff
@@ -209,26 +220,54 @@ module.exports = {
 			await enableSerialConsole(this.context.get().os.image.path);
 		}
 
+
+		this.log("Logging into balena with balenaSDK");
+		await this.context
+		  .get()
+		  .cloud.balena.auth.loginWithToken(this.suite.options.balena.apiKey);
+		await this.context
+		.get()
+		.cloud.balena.models.key.create(
+			this.context.get().sshKeyLabel,
+			keys.pubKey
+		);
+		this.suite.teardown.register(() => {
+			return Bluebird.resolve(
```

Comment:
Is there a reason to add an import and use Bluebird here instead of ES6 `Promise.resolve()`?